integrations/docker/images/chip-build-esp32/Dockerfile

```diff
@@ -10,16 +10,8 @@ RUN set -x \
     && : # last line
 
 RUN set -x \
-    && git clone --depth 1 --recursive -b v4.3 https://github.com/espressif/esp-idf.git /tmp/esp-idf \
-    && : # last line
-
-# TODO: Remove this patch once https://github.com/espressif/esp-idf/pull/7632 is available
-COPY 0001-esp_crt_bundle-remove-EC-ACC-certificate.patch /tmp/esp-idf/0001-esp_crt_bundle-remove-EC-ACC-certificate.patch
-
-WORKDIR /tmp/esp-idf
-RUN set -x \
-    && git apply /tmp/esp-idf/0001-esp_crt_bundle-remove-EC-ACC-certificate.patch \
-    && rm -f /tmp/esp-idf/0001-esp_crt_bundle-remove-EC-ACC-certificate.patch \
+    && git clone --depth 1 --recursive -b release/v4.4 https://github.com/espressif/esp-idf.git /tmp/esp-idf \
+    && git -C /tmp/esp-idf checkout 6a7d83af1984b93ebaefa7ca9be36304806c3dc8 \
```

Comment:
@gjc13 I didn't quite understand this. So, basically you are using a fixed commit on `release/v4.4` branch here. Although, it will be fine for the tests, but people can still use any commit on that branch which may lead to issues?

Reply:
@dhrishi The fixed commit is for CI, to make it consistent and stable.
In the readme files, we use release/v4.4 branch, since v4.4 tag is not ready yet, can switch to v4.4 tag when it's ready.

Reply:
Thanks Shu. That makes sense.